client/landing/subscriptions/components/tab-views/sites/sites.tsx

```diff
@@ -52,6 +60,18 @@ const Sites = () => {
 						searchIcon={ <SearchIcon size={ 18 } /> }
 						onSearch={ handleSearch }
 					/>
+
+					<SelectDropdown
+						className="subscriptions-manager__filter-control"
+						options={ availableFilterOptions }
+						onSelect={ ( selectedOption: Option ) =>
+							setFilterOption( selectedOption.value as SiteSubscriptionsFilterBy )
+						}
+						selectedText={
+							translate( 'View: ' ) + getFilterLabel( availableFilterOptions, filterOption )
```

Comment:
Is this something where we'd use a placeholder for the filter label, so the position in the string can be changed in translations?

Reply:
Good point! I'll change it 👍